### PR TITLE
feat: :sparkles: add privacy and policy files

### PR DESCRIPTION
**Pull request by eudycss**

### CODE REVIEW

File: apps/landing/src/layouts/Layout.astro [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+// src/layouts/Layout.astro
```
Comment (Disble) [inline apps/landing/src/layouts/Layout.astro:2]: Remove comments

File: apps/landing/src/layouts/Layout.astro [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="description" content="Astro description" />
+		<meta name="viewport" content="width=device-width" />
+		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+		<title>{title}</title>
+	</head>
+	<body>
+		<main>
+			<!-- El contenido dinámico se inserta aquí -->
```
Comment (Disble) [inline apps/landing/src/layouts/Layout.astro:21]: Same here

File: apps/landing/src/layouts/Layout.astro [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+			<slot />
+		</main>
+	</body>
+</html>
+
+<!-- Estilos Globales -->
```
Comment (Disble) [inline apps/landing/src/layouts/Layout.astro:27]: Same here

**Comment by Disble** [comment]
Use one type of indentation and format the code.

File: apps/landing/src/pages/privacypolicy.astro [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+<h2>Contact Us</h2>
+<p>If you have any questions about this Privacy Policy, You can contact us:</p>
+
+		<p class="instructions">
+		  By email:  <code>contact@ellectornovato.com</code><br/>	 
+			
+		</p>
+		<ul role="list" class="link-card-grid">
+			<Card
+				href="/termsandconditions"
```
Comment (Disble) [inline apps/landing/src/pages/privacypolicy.astro:202]: This can be import from a const. The const variable need to be create apart.

**Comment by Disble** [comment]
The name of the file Layout is so much generic, please be more specific.

**Comment by Disble** [comment]
Change the titles of the pages (browser).

File: apps/landing/src/layouts/Layout.astro (hunk 1 above)
Comment (eudycss) [reply apps/landing/src/layouts/Layout.astro:2]: Without these delimiters(---), the file will not be able to process the code correctly.

File: apps/landing/src/layouts/Layout.astro (hunk 3 above)
Comment (eudycss) [reply apps/landing/src/layouts/Layout.astro:27]: Done

File: apps/landing/src/layouts/Layout.astro (hunk 2 above)
Comment (eudycss) [reply apps/landing/src/layouts/Layout.astro:21]: done

**Comment by eudycss** [comment]
I have finished making all the corrections

**Pull request closed without merging**